### ISSUE TITLE
Apollo 0.9.5 supports Xcode 10.2 beta (10P82S)

### ISSUE THREAD

**Issue by ivanhoe** [apollographql/apollo-ios#451]
Hi guys im trying to create a new project from scratch using the last version of the  pod.
the next one is my podfile
**Pod:**

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'xcode_10_2' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for xcode_10_2
  pod  'Apollo'

end
```

Now the podifile.lock

**Pod.lock:**
```
PODS:
  - Apollo (0.9.5):
    - Apollo/Core (= 0.9.5)
  - Apollo/Core (0.9.5)

DEPENDENCIES:
  - Apollo

SPEC REPOS:
  https://github.com/cocoapods/specs.git:
    - Apollo

SPEC CHECKSUMS:
  Apollo: 19caf3246d292bbb69ecad1bd098661f78222aba

PODFILE CHECKSUM: b3d9f7240a12ba49627382904778b1ed996ef547

COCOAPODS: 1.5.3
```

When im trying to compile the project the next exception is thrown

![screen shot 2019-02-01 at 8 10 20 pm](https://user-images.githubusercontent.com/138103/52158412-c460c100-265d-11e9-9fd2-c9b16f3e1999.png)

The compiler by default is set to swift 4.2 for the project target 

![screen_shot_2019-02-01_at_8_14_55_pm](https://user-images.githubusercontent.com/138103/52158475-59fc5080-265e-11e9-9d91-2f1dd4a4ae6c.png)

and for the pods

![pods](https://user-images.githubusercontent.com/138103/52158538-d7c05c00-265e-11e9-8a01-b2d773d4f0f9.png)

Am im missing something guys?

**Comment by bachand** [comment]
This seems to be resolved on `master`. I am fairly certain https://github.com/apollographql/apollo-ios/commit/b28c3dcae50c36518bc75036948eda91aa54d6ec is what fixed this.

@martijnwalraven maybe we can bump the version to allow people to develop with Xcode 10.2 more easily?

**Comment by ivanhoe** [comment]
Hi @bachand so getting the master branch we could compile =D ?

**Comment by bachand** [comment]
Yes. You could consider pinning your Apollo to the most recent SHA on `master` until a new version is released.

**Comment by ivanhoe** [comment]
thanks so much @bachand!!

**Comment by designatednerd** [comment]
New versions that support Swift 5 and Xcode 10.2 have been released! Please update to anything > 0.11.1. Thanks, and I'll close this one out.
